rush-plugins/rush-http-build-cache-plugin/src/schemas/plugin-config.schema.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Configuration for build cache with HTTPS server",
+  "type": "object",
+  "required": ["url"],
+  "properties": {
+    "url": {
+      "type": "string",
+      "description": "(Required) The URL of the server that stores the caches (e.g. \"https://build-caches.example.com\").",
+      "format": "uri"
+    },
+    "method": {
+      "type": "string",
+      "description": "(Required) The URL of the server that stores the caches (e.g. \"https://build-caches.example.com\").",
+      "enum": ["PUT", "POST", "GET", "PATCH"],
```

```suggestion
      "enum": ["PUT", "POST", "PATCH"],
```

`GET` can't take a body.